features/i18n.feature

```diff
@@ -108,25 +108,27 @@ Feature: Pro supports multiple languages
     Given a `<release>` `<machine_type>` machine with ubuntu-advantage-tools installed
     # # Change the locale
     When I apt install `language-pack-fr`
-    And I run `update-locale LANG=fr_FR.UTF-8` with sudo
+    And I run `locale-gen fr_FR` with sudo
+    And I run `update-locale LANG=fr_FR` with sudo
     And I reboot the machine
     And I run `cat /etc/default/locale` as non-root
-    Then stdout matches regexp:
+    Then I will see the following on stdout:
       """
-      LANG=fr_FR.UTF-8
```

Comment:
This test is failing due to:
```
      Assertion Failed: Expected to find exactly:                                                                                                                                                                  
        LANG=fr_FR                                                                                                                                                                                                 
      But got:                                                                                                                                                                                                     
        #  File generated by update-locale                                                                                                                                                                         
        LANG=fr_FR 
```

Reply:
I had to change the whole thing, see below